src/WebInterface.hs

```diff
@@ -136,6 +144,13 @@ viewProject info state =
 
     viewProjectQueues info state
 
+viewOwner :: [(ProjectInfo, ProjectState)] -> Html
+viewOwner = mapM_ (uncurry viewProjectListItem)
```

Comment:
Now we just concatenate the project pages. I think it would be nicer to concatenate by header instead, so there would only be one section “Building”, one section “Awaiting approval”, etc. And for each section, we list the things from every project. What do you think?

Reply:
That would be nice, although we need to make sure that it is clear from that context from which repository the PR is coming, as that might not always be clear from the title, like: 
```
repo_name: This is my PR name
other_repo_name: This is another PR
```

Reply:
Yes, agreed. Not sure what the best way to go is here. We could add prefixes, but that is repetitive and eats up horizontal space. Maybe we can emit one `<p>` with the name of the repository as header?
![awaiting-approval-mock](https://user-images.githubusercontent.com/506953/72899337-1777cc00-3d26-11ea-8034-9d5302b9cbfc.png)

Reply:
So basically moving the repository name inside the status as opposed to outside (as it is now).

Reply:
Yep, I think that would work.